srcpkgs/gimp/template

```diff
@@ -19,15 +19,15 @@ maintainer="fosslinux <fosslinux@aussies.space>"
 license="GPL-3.0-only"
 homepage="https://www.gimp.org"
 distfiles="https://download.gimp.org/pub/gimp/v${version%.*}/gimp-${version}.tar.bz2"
-checksum=3f15c70554af5dcc1b46e6dc68f3d8f0a6cc9fe56b6d78ac08c0fd859ab89a25
+checksum=84004642d351b398a4293cd7fd3592044a944f05bb52850ee6068f247c657aa3
 python_version=2
 lib32disabled=yes
 no_generic_pkgconfig_link=yes
 
 pre_configure() {
 	NOCONFIGURE=1 autoreconf -fi
 	if [ "$CROSS_BUILD" ]; then
-		vsed -i 's:^py_prefix=`:py_prefix='"$XBPS_CROSS_BASE"'`:' configure
+		vsed -i 's:^py_prefix=$(:py_prefix='"$XBPS_CROSS_BASE"'):' configure
```

Comment:
this is a false positive from xlint, you can avoid it by using `\x60` instead of the backticks